src/sections/blog-post.liquid

```diff
@@ -0,0 +1,184 @@
+<div class="page-width">
+  {% if section.settings.title != blank %}
+    <header class="section-header text-center">
+      <h2>{{ section.settings.title | escape }}</h2>
+    </header>
+  {% endif %}
+
+  {%- assign blog = blogs[section.settings.blog] -%}
+
+  {% unless blog.articles_count == 0 %}
+    <div class="grid grid--uniform grid--blog">
+      {% for article in blog.articles limit: section.settings.post_limit %}
```

Comment:
Actually this can be simplified. We can remove the above `{% unless %}` statement and use this `{% for ... %}` statement with an [`{% else %}` tag](https://help.shopify.com/themes/liquid/tags/iteration-tags#else)

Reply:
I tried it but I realized we need a parent container to wrap the items when we show as a grid so unfortunately I can't do it :( Unless there's another solution I'm not aware of.